retroguard/src/main/java/com/yworks/yguard/obf/classfile/ClassConstants.java

```diff
@@ -122,6 +122,8 @@
     // new in java 11
     public static final String ATTR_NestHost = "NestHost";
     public static final String ATTR_NestMembers = "NestMembers";
+    // source debug for kotlin
+    public static final String ATTR_KotlinSourceDebugExtension = "SourceDebugExtension";
```

Comment:
Hey @KyongSik-Yoon I just looked up that this is not only reserved to Kotlin but also defined in the general JVM specification. As such it must have slipped when implementing `AttrInfo`. Can you adjust the name of the variable to `ATTR_SourceDebugExtension`? Then it should be good to go!

Reply:
> Hey @KyongSik-Yoon I just looked up that this is not only reserved to Kotlin but also defined in the general JVM specification. As such it must have slipped when implementing `AttrInfo`. Can you adjust the name of the variable to `ATTR_SourceDebugExtension`? Then it should be good to go!
> 

ok, I'll do that.

Reply:
@Fohlen Constant name changed to SourceDebugExtension. 
Check it please.